Doc/library/datetime.rst

```diff
@@ -287,11 +287,12 @@ Supported operations:
 |                                | ``[D day[s], ][H]H:MM:SS[.UUUUUU]``, where D  |
 |                                | is negative for negative ``t``. (5)           |
 +--------------------------------+-----------------------------------------------+
-| ``repr(t)``                    | Returns a string in the form                  |
-|                                | ``datetime.timedelta(D[, S[, U]])``, where D  |
-|                                | is negative for negative ``t``. (5)           |
+| ``repr(t)``                    | Returns a string representation of the        |
+|                                | :class:`timedelta` object such that           |
+|                                | ``t == eval(repr(t))``.                       |
 +--------------------------------+-----------------------------------------------+
```

Comment:
`t == eval(repr(t))` isn't necessarily true (e.g. when `datetime` is aliased during import). So this part of the documentation would need to change again.